app/lib/summary_stats_utils.rb

```diff
@@ -88,4 +88,111 @@ def self.ingest_run_count(start_date: Time.zone.today, end_date: Time.zone.today
+  # the 'updates' attribute is a hash of properties to counts of times that property was updated
+  # Updates to study files are tracked as a 'file updates' property in that hash
+  # by default, this excludes reporting on studies that have been created/deleted within the time period
+  def self.updated_studies_info(start_date: Time.zone.today, end_date: Time.zone.today + 1.day, exclude_create_delete: true)
+    updates = HistoryTracker.trackers_by_date(Study, action: 'update', start_time: start_date, end_time: end_date).to_a
+    excluded_ids = []
+    # we typically exclude created and deleted studies from this report, since those are handled separately
+    # however allowing their inclusion can help for testing
+    if exclude_create_delete
+      creates_and_deletes = HistoryTracker.where(scope: 'study', :created_at.gt => start_date, :created_at.lt => end_date, :action.in => ['create', 'delete'])
+      excluded_ids = creates_and_deletes.map{ |tracker| tracker.association_chain.first['id'] }
+    end
+
+    # for each study id, assemble a hash of property names to # of times they've been modified
+    updates_by_id = {}
+    updates.each do |update|
+      study_id = update.association_chain.first['id'].to_s
+      if excluded_ids.to_s.exclude?(study_id)
+        updates_by_id[study_id] ||= {}
+        update['modified'].keys.each {|key| updates_by_id[study_id][key] = updates_by_id[study_id][key].to_i + 1 }
+      end
+    end
+
+    # now update the hash with the number of times a study file has been updated
+    study_file_updates(excluded_ids, updates_by_id, start_date: start_date, end_date: end_date)
+
+    update_info = updates_by_id.map do |id, value|
+      study = Study.find(id)
+      if study.present?
+        {
+          title: study.name,
+          study_owner: study.user.try(:email),
+          accession: study.accession,
+          updates: value
+        }
+      else
+        nil
+      end
+    end.compact
+    update_info
+  end
+
+  # returns a hash of study ids to the number of file updates performed
+  # this number is stored in a "file updates" property on the hash, so it can be merged with other
+  # update properties collected and passed in via the updates_by_id argument
+  def self.study_file_updates(excluded_study_ids, updates_by_id={}, start_date: Time.zone.today, end_date: Time.zone.today + 1.day)
+    file_updates = HistoryTracker.trackers_by_date(StudyFile, start_time: start_date, end_time: end_date).to_a
+    file_updates.each do |update|
+      study_file_id = update.association_chain.first['id'].to_s
+      study_file = StudyFile.find_by(id: study_file_id)
+      study_id = nil
+      if study_file.present?
+        study_id = study_file.study_id.to_s
+      elsif update['action'] == 'destroy'
+        study_id = update['original']['study_id'].to_s
+      end
+      if study_id.present? && excluded_study_ids.to_s.exclude?(study_id)
+        updates_by_id[study_id] ||= {}
```

Comment:
A possible refinement
```suggestion
        updates_by_id[study_id] ||= {'file updates' => 0}
        updates_by_id[study_id]['file updates'] += 1
```